src/lib/stdbigos/string.c

```diff
@@ -105,69 +111,82 @@ int memcmp(const void* lhs_, const void* rhs_, size_t n) {
 void* memchr(const void* src_, int ch, size_t n) {
 	const u8* s = src_;
 
-	while(n--) {
-		if(*s == ch) return (void*)s;
+	while (n--) {
+		if (*s == ch)
+			return (void*)s;
 		s++;
 	}
 
 	return nullptr;
 }
 
 int strcmp(const char* lhs, const char* rhs) {
-	while(true) {
-		if(*lhs < *rhs) return -1;
-		if(*lhs > *rhs) return 1;
-		if(!*lhs) return 0;
+	while (true) {
+		if (*lhs < *rhs)
+			return -1;
+		if (*lhs > *rhs)
+			return 1;
+		if (!*lhs)
+			return 0;
 		lhs++;
 		rhs++;
 	}
 }
 
 int strncmp(const char* lhs, const char* rhs, size_t n) {
-	while(n--) {
-		if(*lhs < *rhs) return -1;
-		if(*lhs > *rhs) return 1;
-		if(!*lhs) return 0;
+	while (n--) {
+		if (*lhs < *rhs)
+			return -1;
+		if (*lhs > *rhs)
+			return 1;
+		if (!*lhs)
+			return 0;
 		lhs++;
 		rhs++;
 	}
 	return 0;
 }
 
 char* strchr(const char* str, int ch) {
-	while(true) {
-		if(*str == ch) return (char*)str;
-		if(!*str) return nullptr;
+	while (true) {
+		if (*str == ch)
+			return (char*)str;
+		if (!*str)
+			return nullptr;
 		str++;
 	}
 }
 
 char* strrchr(const char* str, int ch) {
 	char* res = nullptr;
 
-	while(*str) {
-		if(*str == ch) res = (char*)str;
+	while (*str) {
+		if (*str == ch)
+			res = (char*)str;
 		str++;
```

Comment:
nit: not certain whether I like this formatting, perhaps a newline after brace-less `if` would help?  
But there was no newline before pr so I guess it is not in the scope of this one.